charts/rancher-k3s-upgrader/0.3.0/templates/configmap.yaml

```diff
@@ -8,7 +8,7 @@ data:
   SYSTEM_UPGRADE_CONTROLLER_THREADS: {{ .Values.systemUpgradeControllerThreads | default "2" | quote }}
   SYSTEM_UPGRADE_JOB_ACTIVE_DEADLINE_SECONDS: {{ .Values.systemUpgradeJobActiveDeadlineSeconds | default "900" | quote }}
   SYSTEM_UPGRADE_JOB_BACKOFF_LIMIT: {{ .Values.systemUpgradeJobBackoffLimit | default "99" | quote }}
-  SYSTEM_UPGRADE_JOB_IMAGE_PULL_POLICY: {{ .Values.systemUpgradeJobImagePullPolicy | default "Always" | quote }}
+  SYSTEM_UPGRADE_JOB_IMAGE_PULL_POLICY: {{ .Values.systemUpgradeJobImagePullPolicy | default "IfNotPresent" | quote }}
```

Comment:
Could this change have negative implications for upgrade scenarios, cleaned nodes, etc.?

Reply:
Can't think of any, unless image is force pushed or something and tag remains the same. Anything specific scenario standing out for you?

Reply:
`IfNotPresent` is the usual default for a pull policy. If the image doesn't exist, then pull it, but always pulling breaks airgap setups. 

If you have a more specific example of where you think this could be a problem then we can answer more directly.

Reply:
I had originally thought that might need to pull again, but `IfNotPresent` covers this. Resolving